PyInstaller/utils/hooks/__init__.py

```diff
@@ -65,29 +65,36 @@ def __exec_python_cmd(cmd, env=None):
         pp = os.pathsep.join([pp_env.get('PYTHONPATH'), pp])
     pp_env['PYTHONPATH'] = pp
 
-    txt = exec_python(*cmd, env=pp_env)
-    return txt.strip()
+    if ret_stdout:
+        txt = exec_python(*cmd, env=pp_env)
+        return txt.strip()
+    else:
+        return exec_python_rc(*cmd, env=pp_env)
+
+
+def __exec_statement(statement, ret_stdout=True):
+    statement = textwrap.dedent(statement)
+    cmd = ['-c', statement]
+    return __exec_python_cmd(cmd, ret_stdout=ret_stdout)
 
 
 def exec_statement(statement):
     """
     Executes a Python statement in an externally spawned interpreter, and
     returns anything that was emitted in the standard output as a single string.
     """
-    statement = textwrap.dedent(statement)
-    cmd = ['-c', statement]
-    return __exec_python_cmd(cmd)
+    return __exec_statement(statement, ret_stdout=True)
```

Comment:
Nitpick, maybe not needed to set the argument as it is its default value? If you did it for clarity purpose, then good :)
```suggestion
    return __exec_statement(statement)
```

Reply:
I would say the same for `exec_script()`.

Reply:
Yeah, the idea is for the stdout and rc versions of the call to be visually similar. I think it makes the intent more clear, as well.